### ISSUE TITLE
ext option ignored

### ISSUE THREAD

**Issue by marco-fp** [getsentry/sentry-webpack-plugin#8]
```
    new SentryCliPlugin({
      release: hash => hash,
      include: `${paths.build}`,
      ext: 'css',
      ignore: ['.'],
      configFile: 'sentry.properties',
    }),
```

Does not upload my `.css` bundle, same happens with `ext:'.css'`

Am I doing anything wrong? 

I can confirm that the .css file is there, same directory as the rest of bundles.

**Comment by jardakotesovec** [comment]
I am currently running to similar problems. There are only few of the advertised options actually handled and passed to sentry-cli.
In my case I am missing `urlPrefix` to be passed to sentry-cli.

**Comment by montogeek** [comment]
I implemented all options on this PR https://github.com/getsentry/sentry-webpack-plugin/pull/6 and in the CLI itself https://github.com/getsentry/sentry-cli/pull/182
We should just wait for the next release

**Comment by jardakotesovec** [comment]
@montogeek I see, thanks for that! I guess we can close this.

**Comment by jan-auer** [comment]
`sentry-cli 1.27.0` has been released and dependency here updated in 23ad4d95. Let me have a look at the other issues quickly and then we will issue a release of this plugin. Sorry that it took so long this time and thank you for your patience!

**Comment by jan-auer** [comment]
Released v1.1.1
